src/status_im/translations/af.cljs

```diff
@@ -148,10 +148,6 @@
    :chat-send-eth                         "{{amount}} ETH"
    :chat-send-eth-to                      "{{amount}} ETH aan {{chat-name}}"
    :chat-send-eth-from                    "{{amount}} ETH van {{chat-name}}"
-   :command-text-location                 "Ligging {{address}}"
```

Comment:
Hm, I can find  `location_title` in js file, but I didn't find any analog of `command-text-send`, or we don't need this translation?